.travis.yml

```diff
@@ -19,7 +19,10 @@ before_install:
   - sudo make base
   - sudo make apt-openfoam-deps apt-simphony-deps apt-lammps-deps apt-mayavi-deps apt-aviz-deps
   - make fix-pip
-  - pip install --allow-external PIL --allow-unverified PIL PIL
+  - sudo pip uninstall -y argparse
```

Comment:
correction in this case `sudo pip -install --upgrade argparse` in `fix-pip` should be the right thing to do. Maybe rename `fix-pip` -> `fix-system-python`